website/docs/d/iploadbalancing_failover_ip_attach_html.markdown

```diff
@@ -0,0 +1,27 @@
+---
+subcategory : "Load Balancer (IPLB)"
+---
+
+# ovh_iploadbalancing_vrack_network (Data Source)
```

Comment:
What is the correct name of the documetation page? :-) 

"ovh_iploadbalancing_failover_ip_attach" ? ;-)